### PR TITLE
Allow signing wrapped segwit inputs with missing non-witness-utxo

### PR DESCRIPTION
**Pull request by bigspider**
The app allows signing native segwit inputs with missing non-witness-utxo in the PSBT (since version 2.0.6). However, the logic to detect the condition is based on the input's scriptPubKey, rather than the wallet policy. Therefore, wrapped segwit inputs are outright rejected if the non-witness-utxo is missing.

This PR makes the behavior of the app more consistent: for wrapped segwit inputs, a warning is given if the non-witness-utxo is missing, but signing is otherwise still allowed.

Closes: #127

### CODE REVIEW

File: tests/automations/sign_with_wallet_missing_nonwitnessutxo_accept.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "version": 1,
+  "rules": [
+    {
+      "regexp": "Unverified|Update|or third party|[S]?pend from|Wallet name|Review|Amount|Address|Confirm|Fees",
+      "actions": [
+        ["button", 2, true],
```
Comment (sgliner-ledger) [inline tests/automations/sign_with_wallet_missing_nonwitnessutxo_accept.json:7]: This will require changes to work with Stax :)

**Review by sgliner-ledger — APPROVED** [review]

File: tests/automations/sign_with_wallet_missing_nonwitnessutxo_accept.json (hunk 1 above)
Comment (bigspider) [reply tests/automations/sign_with_wallet_missing_nonwitnessutxo_accept.json:7]: Indeed, now I remember that that's why I left the PR as WIP before...

File: src/common/wallet.c [hunk 2]
```diff
@@ -1876,6 +1876,24 @@ int parse_descriptor_template(buffer_t *in_buf, void *out, size_t out_len, int v
     return parse_script(in_buf, &out_buf, version, 0, 0);
 }
 
+int get_policy_segwit_version(const policy_node_t *policy) {
```
[the remark this answers is not in the document]
Comment (bigspider) [reply src/common/wallet.c:1879]: Technically, the segwit version byte is a number between 0 and 16 (except that only versions 0 and 1 are defined at this time).
Also, there are two different ways of returning 0, so I'd still need a comment there.

**Comment by bigspider** [comment]
Rebased on top of #126, and added 2f2cda0bf4a2e4fad42c177e825939bc41c43fba for the Stax automation.

**Review by sgliner-ledger — APPROVED** [review]

**Pull request merged**